simple-junit/src/test/java/com/hmkcode/junit/MathTest.java

```diff
@@ -20,44 +19,48 @@
 @RunWith(JUnit4.class)
 public class MathTest {
 
-   @Rule
-   public Timeout globalTimeout = new Timeout(3000); // 3 seconds max per method tested
+    @Rule
+    public Timeout globalTimeout = new Timeout(3000); // 3 seconds max per method tested
+
+    private Math math;
+
+    @Before
+    public void setup() {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [You may have misspelled a JUnit framework method (setUp or tearDown)](https://www.codacy.com/app/maratmingazovr/Java/file/2724646881/issues/source?bid=3348317&fileBranchId=3348322#l28)